docs/docs/configuration.md

```diff
@@ -39,52 +39,53 @@ Iceberg tables support table properties to configure table behavior, like the de
 
 ### Write properties
 
-| Property                                             | Default                     | Description                                                                                                                                                                                       |
```

Comment:
looks like this is auto-formatted by IDE, can we minimizes the changes to only the new configs? If there is not a good way, I guess we will have to commit the change like this.